clinica/iotools/converter_utils.py

```diff
@@ -1,14 +1,26 @@
-def sort_session_list(session_list):
-    session_idx = [int(session[5:]) for session in session_list]
-    session_idx.sort()
-    session_id_list = []
-    for session in session_idx:
-        if session < 10:
-            session_id_list.append(f"ses-M0{session}")
-        else:
-            session_id_list.append(f"ses-M{session}")
+from typing import List
+
+
+def sort_session_list(session_list: List[str]) -> List[str]:
+    """Sorts the list of session IDs provided based on their session number.
 
-    return session_id_list
+    Parameters
+    ------------
+    session_list : list[str]
+        List of session IDs to sort.
+
+    Returns
+    --------
+    list[str] :
+        Sorted list of session IDs.
```

Comment:
Could you take a few of the unit tests you wrote and put them in an Examples section here ? This is cheap to do and might help (some) readers understand why this function exists over the built-in `sorted`.